### ISSUE TITLE
Change the ring key stroke to nvda+alt+up and down arrow and give the user the possibility to change it in the input gesture dialog

### ISSUE THREAD

**Issue by Adriani90** [munawarb/NVDA-Time-Zoner#5]
Steps to reproduce
1. Navigate the ring with nvda+alt+t

Actual behavior: NVDA can navigate the ring only in one direction and the interval between presses is quite short.

Expected behavior: the ring should be consistent with the synth settings ring for example. Wit two scripts (nvda+alt+up and nvda+alt+down arrow), users could navigate the ring in both directions and the interval between presses is irrelevant.
I would also expect users to be able to reassign the both key strokes (i.e. to nvda+alt+left arrow and nvda+alt+right arrow or something else).

**Comment by JessicaTegner** [comment]
Even better, would be to include it as part of the normal NVDA Ring (I think it's called).
The thing you get when you (on the labtop layout) press NVDA + SHIFT + CTRL + An arrow key.

**Comment by munawarb** [comment]
I like that suggestion. So you press NVDA+CTRL+SHIFT+UP/DOWN to select the timezone. Then you can press NVDA+ALT+T to hear that timezone any time.

If you press NVDA+ALT+T rapidly it will cycle through the timezones from that point in the ring.

I'll have to see how to add things to the NVDA ring.

**Comment by Adriani90** [comment]
this is the synth settings ring and is designed for synthesizer settings. But I agree the synth settings ring should be flexible so that we can add or remove elements. but this is out of scope here in my view.

Von meinem iPhone gesendet

**Comment by Adriani90** [comment]
i would say report time zone during the sellection in the ring as well.

Von meinem iPhone gesendet
